www/src/examples/Badge.js

```diff
@@ -1,7 +1,3 @@
-const badgeInstance = (
-  <p>
-    Badges <Badge>42</Badge>
-  </p>
-);
-
-render(badgeInstance);
```

Comment:
I figure we might as well use the shorthand syntax everywhere. We're already using it in many places, anyway.

Reply:
👍 ya definitely i wanted to do that but didn't have time